### DISCUSSION TITLE
How can I disable Sunday and past days?

### DISCUSSION THREAD

**Discussion by ashwyan** [quasarframework/quasar#14970] [General - Components / Directives / etc] [answered]
I have calendar and time, but I want to disable Sunday and past date from today. How can I do it?

**Comment by metalsadman** [comment, marked as answer]
see this example https://quasar.dev/vue-components/date#example--options and provide your own filter logic. another example from this thread https://github.com/quasarframework/quasar/discussions/14930#discussioncomment-4176639
